tests/stress/readme.md

```diff
@@ -103,6 +103,24 @@ Sample command line invocations:
 - Extension restart enabled.
 - Delay of 250 ms between successive extension restarts.
 
+## 1.6. bindmonitor_tailcall_invoke_program_test
+This test first loads a specific native eBPF program. It then loads all the 32 tail call programs and updates the program array map. It then creates the specified # of threads where each thread attempts a TCP 'bind' to the port. The test set up ensures that the `thread_index` passed in each `thread_context` is unique to that thread.
```

Comment:
```suggestion
This test first loads a specific native eBPF program. It then loads all the MAX_TAIL_CALL_CNT tail call programs and updates the program array map. It then creates the specified # of threads where each thread attempts a TCP 'bind' to the port. The test set up ensures that the `thread_index` passed in each `thread_context` is unique to that thread.
```
avoid hard coding the number 32 in other places.  It might change in the future and we want to minimize the number of places to change in such a case since they're easy to miss.